frontend-react/src/components/DataDashboard/ReportDetails/ReportDetailsSummary.module.scss

```diff
@@ -0,0 +1,14 @@
+.ReportDetailsSummary {
```

Comment:
Can we use the USWDS colors / color tokens anywhere we can? https://designsystem.digital.gov/utilities/color/